src/stories/static_data/contactData.js

```diff
@@ -1,5 +1,4 @@
 const contact = {
-  "name":"311",
```

Comment:
the location object has name defined -- which actually corresponds to the location. Not sure what this top level name is for.